reproject/conftest.py

```diff
@@ -1,50 +1,19 @@
 # This file is used to configure the behavior of pytest when using the Astropy
 # test infrastructure.
 import os
-from distutils.version import LooseVersion
-from astropy.version import version as astropy_version
-
-if LooseVersion(astropy_version) < LooseVersion('2.0.3'):
-    # Astropy is not compatible with the standalone plugins prior this while
-    # astroquery requires them, so we need this workaround. This will mess
-    # up the test header, but everything else will work.
-    from astropy.tests.pytest_plugins import (PYTEST_HEADER_MODULES,
-                                              TESTED_VERSIONS)
-elif astropy_version < '3.0':
-    # With older versions of Astropy, we actually need to import the pytest
-    # plugins themselves in order to make them discoverable by pytest.
-    from astropy.tests.pytest_plugins import *
-else:
-    # As of Astropy 3.0, the pytest plugins provided by Astropy are
-    # automatically made available when Astropy is installed. This means it's
-    # not necessary to import them here, but we still need to import global
-    # variables that are used for configuration.
-    from astropy.tests.plugins.display import PYTEST_HEADER_MODULES, TESTED_VERSIONS
 
+from astropy.tests.plugins.display import PYTEST_HEADER_MODULES, TESTED_VERSIONS
 from astropy.tests.helper import enable_deprecations_as_exceptions
 
 enable_deprecations_as_exceptions()
 
-# Uncomment and customize the following lines to add/remove entries from
-# the list of packages for which version numbers are displayed when running
-# the tests. Making it pass for KeyError is essential in some cases when
-# the package uses other astropy affiliated packages.
-try:
-    PYTEST_HEADER_MODULES['Astropy'] = 'astropy'
-    PYTEST_HEADER_MODULES['astropy-healpix'] = 'astropy_healpix'
-    PYTEST_HEADER_MODULES['Cython'] = 'cython'
-    del PYTEST_HEADER_MODULES['h5py']
-    del PYTEST_HEADER_MODULES['Matplotlib']
-except (NameError, KeyError):  # NameError is needed to support Astropy < 1.0
```

Comment:
Keeping the exception around is generally good to not to be sensitive for upstream changes (or in fact it came up as an issue for one of the packages using another affiliated package). Not critical though as we can easily fix it when it comes up.

Reply:
I agree it might make sense to add back, but let's see what's going to happen with the display plugin first.